src/handler/http/request/alerts/mod.rs

```diff
@@ -52,3 +56,34 @@ impl From<AlertError> for HttpResponse {
         }
     }
 }
+
+/// GetAlert
+#[utoipa::path(
+    context_path = "/api",
+    tag = "Alerts",
+    operation_id = "GetAlert",
+    security(
+        ("Authorization"= [])
+    ),
+    params(
+        ("org_id" = String, Path, description = "Organization name"),
+        ("alert_id" = Ksuid, Path, description = "Alert ID"),
```

Comment:
I think it's great that we can leverage `actix_web` directly for type conformant, but I'm guessing there's no such case where clients would need to generate and provide this ID without getting one from server first?

Even if such case exists, clients would only need to provide a `String` for example, and `actix_web` would do the conversion that client doesn't necessary need to know or understand what `Ksuid` is?

Reply:
> but I'm guessing there's no such case where clients would need to generate and provide this ID without getting one from server first?

That's correct. The idea is that the client will just be passing KSUIDs to the backend that it received from responses from the `ListAlerts` route (or potentially the `CreateAlert` route).

> clients would only need to provide a String for example, and actix_web would do the conversion that client doesn't necessary need to know or understand what Ksuid is?

That's correct.

There's really no reason for the UI to generate its own KSUIDs. Theoretically the UI **could** generate it's own KSUID string and if it did so then Actix will enforce that the KSUID is formatted correctly. However, even if the UI did generate it's own KSUID then the randomness of KSUID pretty much guarantees that the KSUID won't already exist in our DB.